docs/pages/identity-governance/device-trust/intune-integration.mdx

```diff
@@ -0,0 +1,172 @@
+Teleport Web UI. Running it as a standalone service is not currently supported.
+</Admonition>
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx edition="Teleport Enterprise"!)
+
+(!docs/pages/includes/device-trust/prereqs.mdx!)
+- Access to Microsoft Entra ID with Global Administrator or Application Administrator permissions.
+- An active Microsoft Intune subscription with managed devices.
+
+## Step 1/3. Register an application in Microsoft Entra ID
+
+[Create and configure an application](https://learn.microsoft.com/en-us/graph/auth-register-app-v2)
+in Microsoft Entra ID that Teleport will use to access your Intune data:
+
+1. Sign in to the [Microsoft Entra admin center](https://entra.microsoft.com) with an account
+   that has Global Administrator or Application Administrator permissions.
+2. Navigate to Entra ID > App registrations and select New registration.
+3. Configure the application:
+   - Name: enter a descriptive name (e.g., "Teleport Intune Integration").
+   - Supported account types: select "Accounts in this organizational directory only".
+   - Redirect URI: leave this blank and click "Register".
+4. Once created, copy the Application (client) ID from the Overview page. You'll need this
+   when configuring the integration in Teleport.
+
+## Step 2/3. Configure API permissions and create client secret
+
+[Grant the application permission](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-configure-app-access-web-apis#application-permission-to-microsoft-graph)
+to read Intune managed devices:
+
+1. In your registered application, navigate to API permissions > Add a permission.
+2. Select Microsoft Graph > Application permissions.
+3. Search for and add the `DeviceManagementManagedDevices.Read.All` permission.
+4. Click "Grant admin consent" for your organization and confirm.
+
+<Admonition type="note">
+The consent must be granted by a Global Administrator or Privileged Role Administrator. Failure to
+grant this permission will result in 401 Forbidden errors from the Graph API.
+</Admonition>
+
+To [create a client secret](https://learn.microsoft.com/en-us/graph/auth-register-app-v2#option-2-add-a-client-secret):
+
+1. In your registered application, navigate to Certificates & secrets > Client secrets.
+2. Click New client secret.
+3. Add a description and select an expiration period (we recommend setting a reminder to rotate
+   the secret before it expires).
+4. Click "Add" and copy the Value (not the Secret ID).
+
+## Step 3/3. Configure the Intune integration in Teleport
+
+1. In the Web UI, navigate to Add New > Integration and select "Microsoft Intune".
+2. Fill in the required information:
+   - Primary domain or Microsoft Entra tenant ID: Enter your [tenant identifier](https://learn.microsoft.com/en-us/partner-center/account-settings/find-ids-and-domain-names#find-the-microsoft-entra-tenant-id-and-primary-domain-name)
```

Comment:
I would make it an explicit step to visit the Microsoft docs and find your tenant identifier. Otherwise, this seems like it could be a definition link that the user can ignore if know what a tenant identifier is.